variables.tf

```diff
@@ -9,3 +13,257 @@ variable "tags" {
   description = "User-Defined tags"
   type        = map(string)
 }
+
+########################################
+# Bitbucket Vars
+########################################
+
+variable "admin_email" {
+  description = "email address for administrator"
+  type        = string
+}
+
+variable "admin_password" {
+  description = "initial password to use for administrator"
+  type        = string
+}
```

Comment:
I really hate passing this as a var

Reply:
It's only used in situations where you're bootstrapping a brand new instance of Bitbucket. Otherwise it is disregarded. I updated the description to reflect this and also added the ability to leave it blank, in which case a meaningless password will be generated.